addon/components/polaris-action-list/item.js

```diff
@@ -113,13 +115,12 @@ export default class PolarisActionListItem extends Component {
     if (active) {
       cssClasses.push('Polaris-ActionList--active');
     }
+    // TODO #polaris-v5-newDesignLanguage add newDesignLanguage styles
 
     return cssClasses.join(' ');
   }
 
-  @computed('image')
   get imageBackgroundStyle() {
-    let { image: url } = this;
-    return url ? htmlSafe(`background-image: url(${url})`) : '';
+    return htmlSafe(`background-image: url(${this.args.image})`);
   }
```

Comment:
Not sure how this works with glimmer, will this update automatically when `args.image` changes (i.e. are properties under `args` automatically `@tracked`)?

Reply:
yes, arguments are auto-tracked 🕺